src/Compilers/CSharp/Portable/Compiler/MethodBodySynthesizer.cs

```diff
@@ -533,14 +539,13 @@ internal static BoundBlock ConstructDestructorBody(MethodSymbol method, BoundBlo
 
             if ((object)baseTypeFinalize != null)
             {
+                var receiver = new BoundBaseReference(syntax, method.ContainingType) { WasCompilerGenerated = true };
                 BoundStatement baseFinalizeCall = new BoundExpressionStatement(
                     syntax,
                     BoundCall.Synthesized(
                         syntax,
-                        new BoundBaseReference(
-                            syntax,
-                            method.ContainingType)
-                        { WasCompilerGenerated = true },
+                        receiver,
+                        initialBindingReceiverIsSubjectToCloning: Binder.TryReceiverIsSubjectToCloning(receiver),
```

Comment:
>Binder.TryReceiverIsSubjectToCloning(receiver) [](http://example.com/codeflow?start=66&length=46)

I think false is the right value here, we are not going to clone base reference.